connexion/apis/flask_api.py

```diff
@@ -296,7 +297,12 @@ def console_ui_home(self):
         }
         if self.options.openapi_console_ui_config is not None:
             template_variables['configUrl'] = 'swagger-ui-config.json'
-        return flask.render_template('index.j2', **template_variables)
+
+        # Use `render_template_string` instead of `render_template` to circumvent the flask
+        # template lookup mechanism and explicitly render the template of the current blueprint.
```

Comment:
Could be nice to include a link to the relevant flask docs or the github issue as well for full context?

Reply:
Added link to my comment on the issue.